packs/age-of-ashes-bestiary/lesser-dragonstorm.json

```
@@ -16,7 +16,7 @@
                 },
                 "category": null,
                 "description": {
-                    "value": "<p><strong>Trigger</strong> A creature attempts to activate Vengegate</p>\n<hr />\n<p><strong>Effect</strong> The chamber within Alseta's Ring fills with a dragonstorm. Creatures within this area take [[/r {2d8[acid],2d8[cold],2d8[fire],2d8[poison]}]]{10d8 damage}, equally split between acid, cold, electricity, fire, and poison.</p>"
+                    "value": "<p><strong>Trigger</strong> A creature attempts to activate Vengegate</p>\n<hr />\n<p><strong>Effect</strong> The chamber within Alseta's Ring fills with a dragonstorm. Creatures within this area take [[/r {2d8[acid],2d8[cold],2d8[electricity],2d8[fire],2d8[poison]}]]{10d8 damage}, equally split between acid, cold, electricity, fire, and poison.</p>"
```

```suggestion
                    "value": "<p><strong>Trigger</strong> A creature attempts to activate Vengegate</p>\n<hr />\n<p><strong>Effect</strong> The chamber within Alseta's Ring fills with a dragonstorm. Creatures within this area take @Damage[2d8[acid],2d8[cold],2d8[electricity],2d8[fire],2d8[poison]]{10d8 damage}, equally split between acid, cold, electricity, fire, and poison.</p>"
```
Let's at least fix that while we're at it.